jwst/lib/pointing_summary.py

```diff
@@ -26,11 +26,11 @@
 >>> print(t["delta_v1"])
           delta_v1
     -------------------
-    0.13712727164010646
+    0.13712727164010
```

Comment:
This is an output alignment issue Move lines 27-29 4 spaces left and keep the previous value

Reply:
Ah, thanks. @stscieisenhamer ignore my question (below), since Nadia just answered it.